src/app/zap-templates/zcl/data-model/chip/power-topology-cluster.xml

```diff
@@ -14,19 +14,23 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<configurator>
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/app_clusters/PowerTopology.adoc
+Parameters: 
+Git: 0.9.2-summer2025-124-g2b25c360b
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
   <domain name="Measurement &amp; Sensing"/>
-
   <cluster code="0x009C">
-    <domain>Measurement &amp; Sensing</domain>
+    <domain>General</domain>
```

Comment:
Any idea why this change and what `domain` even means?

Reply:
I see that this cluster is inside the "Measurement & Sensing" group in the ZAP UI. Also there is another `domain` element in the XML outside of the `cluster` (line 24) that has the attribute `name` set to "Measurement & Sensing". 

I'm not really sure about the differences between them and several other cluster XMLs also have mismatching values in these parts. I think that maybe @hasty would be more familiar with this from Alchemy side.

Reply:
It seems that the `domain` was taken from the links of the PowerTopology cluster to Ch11_Management which goes to the "General" domain. This is actually overriden in [this PR](https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/11537/files#diff-08fe044e0bbfc8d88d2b65eb3f206d105bc2053020195bebba5e3552ef441456R687) but still not merged.